Detectors/MUON/MCH/Simulation/include/MCHSimulation/Response.h

```diff
@@ -32,39 +32,55 @@ class Response
 
   float getQspreadX() { return mQspreadX; };
   float getQspreadY() { return mQspreadY; };
-  float getChargeSat() { return mChargeSat; };
+  float getFCtoADC() { return mFCtoADC; };
   float getChargeThreshold() { return mChargeThreshold; };
   float etocharge(float edepos);
   double chargePadfraction(float xmin, float xmax, float ymin, float ymax);
   double chargefrac1d(float min, float max, double k2, double sqrtk3, double k4);
-  unsigned long response(float charge);
+  unsigned long response(unsigned long adc);
   float getAnod(float x);
   float chargeCorr();
+  bool aboveThreshold(float charge) { return charge > mChargeThreshold; };
+  float getSigmaIntegration() { return mSigmaIntegration; };
+  bool getIsSampa() { return mSampa; };
+  void setIsSampa(bool isSampa = true) { mSampa = isSampa; };
 
  private:
+  //setter to get Aliroot-readout-chain or Run 3 (Sampa) one
+  bool mSampa = true;
+
   //parameter for station number
   Station mStation;
   //proper parameter in aliroot in AliMUONResponseFactory.cxx
-  const float mQspreadX = 0.144; //charge spread in cm
-  const float mQspreadY = 0.144;
+  float mQspreadX = 0.0; //charge spread in cm
```

Comment:
what's the point of setting those values here in the .h (to invalid values) while they are (correctly) set in the ctor anyway ?

Reply:
@aphecetche 
sorry, no point. it will be removed in the next commit.